### PR TITLE
Update node , dependencies, add errorview & request reference

### PR DESCRIPTION
**Pull request by vademo**

### CODE REVIEW

File: app/views/error.ejs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<%- include('_header') -%>
+<div class="m-alert m-alert--danger u-margin-top-xs u-margin-bottom-xs" style="padding: 1rem;">
```
Comment (TriangleJuice) [inline app/views/error.ejs:2]: Inline styles 💩

File: app/views/error.ejs [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  <h5><%- title %></h5><br/>
```
Comment (TriangleJuice) [inline app/views/error.ejs:3]: Brrr! 💩

File: package.json [hunk 3]
```diff
@@ -1,37 +1,35 @@
...
   "version": "2.0.0",
-  "description": "Astad skeleton",
+  "description": "oauth demo app",
   "private": true,
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
     "dev": "env-cmd -f .env.dev nodemon app.js",
-    "lint": "node_modules/eslint/bin/eslint.js  .",
+    "lint": "eslint  .",
```
Comment (TriangleJuice) [inline package.json:10]: FYI 2 spaties

**Review by TriangleJuice — CHANGES_REQUESTED** [review]

**Review by TriangleJuice — APPROVED** [review]

File: app/views/error.ejs (hunk 2 above)
Comment (vademo) [reply app/views/error.ejs]: 🧻 removed

File: app/views/error.ejs (hunk 1 above)
Comment (vademo) [reply app/views/error.ejs]: 🧻 fixed

**Review by jsebrech — APPROVED** [review]

**Pull request merged**